services/processor/native/repository/_Elections/fix_first_elections.go

```diff
@@ -0,0 +1,77 @@
+// Copyright 2019 the orbs-ethereum-contracts authors
+// This file is part of the orbs-ethereum-contracts library in the Orbs project.
+//
+// This source code is licensed under the MIT license found in the LICENSE file in the root directory of this source tree.
+// The above notice should be included in all copies or substantial portions of the software.
+
+package elections_systemcontract
+
+import (
+	"encoding/hex"
+	"fmt"
+	"github.com/orbs-network/orbs-contract-sdk/go/sdk/v1/safemath/safeuint64"
+	"github.com/orbs-network/orbs-contract-sdk/go/sdk/v1/state"
+)
+
+func _firstElectionFixRewards() {
```

Comment:
this was meant to run one time... do we need it in master?!

Reply:
for now i was asked to leave the code (but it is unreachable)